pkg/kubelet/kuberuntime/logs/logs.go

```diff
@@ -165,12 +164,32 @@ func parseCRILog(log []byte, msg *logMessage) error {
 	return nil
 }
 
+// jsonLog is a log message, typically a single entry from a given log stream.
```

Comment:
note the type this comes from so people don't think we can arbitrarily rename these fields

Reply:
ok, adding a note.

Reply:
even stronger phrasing, as in "this deserializes the json log format returned by docker; do not modify this struct's fields or json serialization"